proselint/checks/uncomparables/misc.py

```diff
@@ -45,11 +45,9 @@
 attention and expects you to have done the same.
 """
 import re
-from proselint.tools import memoize
```

Comment:
If this fixed anything, it's because the results of previous test runs were getting cached and so even once you fixed the problem you were still seeing it fail. I saw the same thing. The fix isn't to remove memoization here, it's to clear the cache/shelf between test runs, which must not be happening correctly.

Reply:
You are absolutely correct, that's something I forgot to re-implement after I took it out during my testing.